packages/cfd/src/Components/cfd-download-container.tsx

```diff
@@ -0,0 +1,205 @@
+                <a href={getPlatformMt5DownloadLink('windows')} target='_blank' rel='noopener noreferrer'>
+                    <Icon icon='IcInstallationWindows' width={138} height={40} />
+                </a>
+                <a href={getPlatformMt5DownloadLink('macos')} target='_blank' rel='noopener noreferrer'>
+                    <Icon icon='IcInstallationMacos' width={138} height={40} />
+                </a>
+                <a href={getPlatformMt5DownloadLink('linux')} target='_blank' rel='noopener noreferrer'>
+                    <Icon icon='IcInstallationLinux' width={138} height={40} />
+                </a>
+            </div>
+            <Text as='p' align='center' size='xxxs' className='cfd-dashboard__download-center--hint'>
+                <Localize i18n_default_text='The MT5 desktop app is not supported by Windows XP, Windows 2003, and Windows Vista.' />
+            </Text>
+        </React.Fragment>
+    );
+};
+
+const MobileDownload = ({ is_dark_mode_on, platform }: TMobileDownloadProps) => {
+    return (
+        <React.Fragment>
+            {platform === CFD_PLATFORMS.DXTRADE && (
+                <h1 className='cfd-dashboard__download-container-heading'>
+                    {localize('Download the Deriv X mobile app')}
+                </h1>
+            )}
+            <div
+                className={classnames({
+                    'cfd-dashboard__download-container-links-icons': isMobile() || platform === CFD_PLATFORMS.MT5,
+                })}
+            >
+                {isMobile() && platform === CFD_PLATFORMS.DXTRADE && (
+                    <React.Fragment>
+                        <Icon
+                            icon={is_dark_mode_on ? 'IcDxtradeDeviceTabletLight' : 'IcDxtradeDeviceTablet'}
+                            width={133}
+                            height={106}
+                        />
+                        <Icon
+                            icon={is_dark_mode_on ? 'IcDxtradeDevicePhoneLight' : 'IcDxtradeDevicePhone'}
+                            width={48}
+                            height={74}
+                        />
+                    </React.Fragment>
+                )}
+                {platform === CFD_PLATFORMS.MT5 && (
+                    <React.Fragment>
+                        <Icon icon='IcMt5DeviceTablet' width={133} height={106} />
+                        <Icon icon='IcMt5DevicePhone' width={48} height={74} />
+                    </React.Fragment>
+                )}
+            </div>
+            <div className='cfd-dashboard__download-container-links-download-buttons'>
+                <a
+                    className='cfd-dashboard__download-center-options--mobile-link'
+                    href={
+                        platform === CFD_PLATFORMS.MT5
+                            ? getPlatformMt5DownloadLink('android')
+                            : getPlatformDXTradeDownloadLink('android')
```

Comment:
This can actually be a method that accepts a string of either `android`, `ios`, `huawei`

Reply:
Changed.